### PR TITLE
Fixes #170

### LINKED ISSUE #170: Add info re lbaasv1 EOSD to Project Index and Support Matrix
LBaaSv1 will reach EOSD (end of software development) on Oct. 1st; no new features will be developed, but bugs will continue to be triaged and fixed until EOSS (end of software support). We need to add the standard verbiage regarding both to the [Project Index](http://f5-openstack-docs.readthedocs.io/en/master/project_index.html) and Support Matrix.

Once a solution article reflecting the development and support policy is live on support.f5.com, we need to provide a link to that from the support matrix.

### PR DESCRIPTION
**Pull request by jputrino**
@mattgreene @jgruber 
#### What issues does this address?
Fixes #170 

#### What's this change do?
adds information re EOSD for LBaaSv1 to project index and support matrix.

#### Where should the reviewer start?
View changes inline below or at http://docs-dev.readthedocs.io/en/kilo/.

### CODE REVIEW

File: docs/releases_and_versioning.rst [hunk 1]
```diff
@@ -3,56 +3,108 @@
...
+      - F5 LBaaSv2 Plugin
+      - BIG-IP®
+      - RHEL
+      - Ubuntu
+    * - Kilo
+      - 7.x
+      - | 11.5.2+
+        | 11.6.x
+        | 12.0.x
+        | 12.1.x
```
Comment (mattgreene) [inline docs/releases_and_versioning.rst]: Copy/paste error.  We never tested and added support for 12.11.x

File: docs/releases_and_versioning.rst [hunk 2]
```diff
@@ -3,56 +3,108 @@
...
+      - 6, 7
+      - 12, 14
+    * - Liberty
+      - 8.x
+      - | 11.5.2+
+        | 11.6.x
+        | 12.0.x
+        | 12.1.x
```
Comment (mattgreene) [inline docs/releases_and_versioning.rst]: Same as previous.

File: docs/releases_and_versioning.rst [hunk 3]
```diff
@@ -3,56 +3,108 @@
...
+      - 6, 7
+      - 12, 14
+    * - Mitaka
+      - 9.x
+      - | 11.5.2+
+        | 11.6.x
+        | 12.0.x
+        | 12.1.x
```
Comment (mattgreene) [inline docs/releases_and_versioning.rst]: Same as previous.

File: docs/releases_and_versioning.rst [hunk 4]
```diff
@@ -3,56 +3,108 @@
...
+    F5 will continue to repair defects and perform maintenance on the F5 OpenStack LBaaS version 1 integration until the Openstack Ocata release in April 2017.
+
+    *The information below is provided for informational purposes only.*
+
+
+.. list-table:: F5® - OpenStack Neutron Compatibility -- LBaaSv1
+    :header-rows: 1
+
+    * - OpenStack release
+      - F5 LBaaSv2 Plugin
```
Comment (mattgreene) [inline docs/releases_and_versioning.rst]: Should read LBaaSv1 (not 2).

**Comment by jputrino** [comment]
@mattgreene -- should be all set now. Thanks!

**Comment by mattgreene** [comment]
This is still wrong.  I only tagged the 3 lbaasv1 references to 12.1 as incorrect.  You removed 12.1 from the lbaasv2 table, but we did test and support 12.1 for lbaasv2.

**Pull request merged**